### ISSUE TITLE
Provide SharpElectrodeSeries

### ISSUE THREAD

**Issue by apdavison** [NeurodataWithoutBorders/nwb-schema#479]
The pynwb.icephys module has PatchClampSeries but no SharpElectrodeSeries. As far as I can see this could just be a copy of PatchClampSeries, since the latter links to IntracellularElectrode, which can represent either a sharp or patch electrode.

**Comment by oruebel** [comment]
CC @t-b

**Comment by bendichter** [comment]
@apdavison is this blocking you? If so, we could create an extension for this, which will be faster than changing the core schema.

**Comment by apdavison** [comment]
no, it's not blocking. We do have recordings with sharp electrodes in EBRAINS, but we haven't yet started systematically converting datasets to NWB, so it's not urgent.

**Comment by t-b** [comment]
@apdavison Do you have a link which explains what a sharp electrode is? I'm one of the developers of [MIES](https://github.com/AllenInstitute/MIES/) which is an icephys acquisition program and I'm always curious to learn about icephys stuff I don't know.

**Comment by apdavison** [comment]
The [Wikipedia article for electrophysiology](https://en.wikipedia.org/wiki/Electrophysiology#Sharp_electrode_recording) gives a reasonable explanation.

A sharp electrode is like a patch electrode but with a smaller pore, and penetrates inside the cell rather than staying at the  membrane surface.
